.github/workflows/upload-to-pypi.yml

```
@@ -13,7 +13,7 @@ env:
   SETUPTOOLS_SCM_PRETEND_VERSION: ${{ github.event.inputs.overrideVersion }}
   # Run the package tests using `pytest`
   CIBW_TEST_REQUIRES: pytest
-  CIBW_TEST_COMMAND: pytest --pyargs alphacsc
+  CIBW_TEST_COMMAND: pytest --pyargs alphacsc -k "not sporco and not kmc2"
```

It seems that wheels are tested with tests that are outside the `alphacsc/tests/` folder (for instance tests from `alphacsc/other/`).

Maybe try
```suggestion
  CIBW_TEST_COMMAND: pytest --pyargs alphacsc/tests
```

---

wdyt @tomMoral

---

Yes removing sporco is a good idea. It is probably ok to remove it.
Would it be okay to do the two changes in this PR?